### DISCUSSION TITLE
Auto-IDLE VM if not in use

### DISCUSSION THREAD

**Discussion by Shaarigan** [winapps-org/winapps#32] [Ideas]
Hey,

I'm interested in your project especially since I look forward to make Linux my every day environment to work on. I came across your project by looking for making Playnite run on Linux (either via WINE/Bottles or by your or a similar project). There was however another project mentioned in the [Playnite-Forum](https://playnite.link/forum/thread-1156-post-8271.html#pid8271) called [cassowary](https://github.com/casualsnek/cassowary) that aims to be more resource saving approach.

> Cassowary supports:
    Running Windows applications as if they were native applications
    Opening files from a Linux host directly inside Windows applications
    Using Linux apps to open files that are on a Windows VM
    Allowing easy access between both the guest and host filesystems
    An easy-to-use configuration utility
    Creating an application launcher for Windows application
    Automatically suspending the VM when no Windows application is in use and automatically resuming it when required (virt-manager only)

I'm especially interested in that *Automatically suspending the VM when no Windows application is in use and automatically resuming it when required (virt-manager only)* feature and want to know if your project supports that as well. I however could't find any information about that in your repo. If so, I'm sorry if I haven't seen them

**Comment by oskardotglobal** [comment]
Winapps is currently undergoing a rewrite. This is definitely a planned feature, but not supported at the moment. However, doing this using legacy winapps and quickemu would probably be really easy, since you could just start the VM before connecting via RDP and making the required changes to the script should be quite straight forward and something we could all assist you with.

**Comment by oskardotglobal** [comment]
Also, unless you're running dual GPUs you'd probably want single-gpu-passthrough, which is also unsupported at the moment.

**Reply by Shaarigan** [reply]
Depends on what this is supposed to offer in the end. I mean I can already run some productivity Apps like Photoshop 2021 on WINE and a ton of games already work thanks to the work of Steam and [GloriousEggroll](https://github.com/GloriousEggroll) so what I'm currently looking this or [cassowary](https://github.com/casualsnek/cassowary) to achieve is to run those Apps that I don't get to run on Linux currently by neither WINE or Proton.

[Playnite](https://github.com/JosefNemec/Playnite) for example (they have a rewrite for Linux on their roadmap but that might take a while) or Visual Studio but we also have Rider as an replacement

**Comment by LDprg** [comment]
This seems like a duplicate of #17

**Comment by flashifloosh** [comment]
Is there any update on this?
